rmgpy/molecule/molecule.py

```diff
@@ -1595,4 +1595,38 @@ def toGroup(self):
         
         return group
 
-
+    def getNthNeighbor(self, startingAtoms, n, blacklist):
+        '''
+        Recursively get the Nth nonHydrogen neighbors of the startingAtoms.
+        StartingAtoms should be a list.
+        Return a list.
+        '''
+        neighbors = []
+        for atom in startingAtoms:
+            for neighbor in self.getBonds(atom):
+                if neighbor.isNonHydrogen():
+                    neighbors.append(neighbor)
+        neighbors = list(set(neighbors)-set(blacklist))
+        for atom in startingAtoms:
+            blacklist.append(atom)
+        if n>1:
+            neighbors = self.getNthNeighbor(neighbors, n-1, blacklist)
+        return neighbors
+
+    def getBaseCb(self, atom):
```

Comment:
I don't understand what this is doing. It returns [atom] if its a Cb atom or a list of neighboring Cb atoms? What does the labeling of atoms do?